CONTRIBUTING.md

```diff
@@ -36,7 +36,7 @@ pre-commit install
 Or run them manually:
 
 ```bash
-pre-commit --all
+pre-commit run
```

Comment:
this runs pre-commit only on staged files and I wouldn't assume that beginner contributors know the difference..

Reply:
actually i don´t know how it worked, I just tried to run it and it didn´t do anything:
```
usage: pre-commit [-h] [-V] {autoupdate,clean,gc,init-templatedir,install,install-hooks,migrate-config,run,sample-config,try-repo,uninstall,validate-config,validate-manifest,help,hook-impl} ...
pre-commit: error: unrecognized arguments: --all
```

Reply:
Oh right, it should say
```suggestion
pre-commit run --all-files
```

(but `pre-commit run --all` works too)